packages/emulation/libretro-fsuae/package.mk

```diff
@@ -28,7 +28,8 @@ pre_configure_target() {
 }
 
 make_target() {
-  make -j1 CC="$CC" gen
+  make -j1 CC="$TOOLCHAIN/bin/host-gcc" CPPFLAGS="$HOST_CPPFLAGS" CFLAGS="$HOST_CFLAGS" LDFLAGS="$HOST_LDFLAGS" gen
```

Comment:
`$TOOLCHAIN/bin/host-gcc` == `$HOST_CC`

Reply:
Replaced by $HOST_CC

Commit updated to 21d5c1a97ed558fb2c578a1b7e3da4c1e609a5f9

Reply:
Commit updated.